frontend/packages/dev-console/src/components/topology/components/edges/TrafficConnector.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from 'react';
-import { Edge, EdgeConnectorArrow } from '@patternfly/react-topology';
+import { Edge, EdgeConnectorArrow, WithContextMenuProps } from '@patternfly/react-topology';
 import { BaseEdge } from './BaseEdge';
 import './TrafficConnector.scss';
 
 type TrafficConnectorProps = {
   element: Edge;
-};
+} & WithContextMenuProps;
 
-const TrafficConnector: React.FC<TrafficConnectorProps> = ({ element }) => (
-  <BaseEdge element={element} className="odc-traffic-connector">
+const TrafficConnector: React.FC<TrafficConnectorProps> = ({ element, ...others }) => (
```

Comment:
Shouldn't need to change this component if `BaseEdge` props are defined properly.